.circleci/config.yml

```diff
@@ -1330,7 +1330,20 @@ commands:
     - run:
         name: Generate Slack Message JSON
         command: |
-          PR_NUMBER=${CIRCLE_PULL_REQUEST##*/}
+          env_name=<< parameters.target_env >>
```

Comment:
- IF event is a pull request, set `PR_NUMBER` to the PR number of the current request
- ELSE IF last commit message is a "Merge..." commit, scrape the set `PR_NUMBER` to the PR number scraped from the commit message
- ELSE, echo "Not found"